### ISSUE TITLE
LG G2 stremio only plays hdr even though file has dolby vision on Same movie or series

### ISSUE THREAD

**Issue by arrow00009** [Stremio/stremio-bugs#840]
**Describe the bug**
For some reason hdr was the only thing thats get selected on viewing a series or movie 

**Expected behavior**
Is there a way that we could select dv over hdr 

**TV (LG G2 ):**
 - Device: [Webos ]
 - OS: [webos 22 ]

**Comment by NickCh11** [comment]
Yes same problem and obviously it's stremio's issue, all other apps handle's dv properly.

**Comment by krebsf01** [comment]
I tried playing a MP4 file and it worked with Dolby Vision instead of HDR, but some have noted that when the file is mkv it won't work.

**Comment by NickCh11** [comment]
> I tried playing a MP4 file and it worked with Dolby Vision instead of HDR, but some have noted that when the file is mkv it won't work.

yep i can confirm the mkv is the issue

**Comment by TRtomasz** [comment]
Outside stremio control
